config/example/config.txt

```diff
@@ -273,4 +273,13 @@ EVENT_DELAY_UPPER 15;45;70
 EVENT_CUSTOM_START_MAJOR 80;100
 
 ## Strength of ambient star light. Set to 0 or less to turn off.
-STARLIGHT 0
\ No newline at end of file
+STARLIGHT 0
+
+## Player rerouting stuff
+## If not 0, players can be rerouted to an overflow server after a certain cap is reached
+
+ ## Cap before players start being rerouted
+PLAYER_REROUTE_CAP 0
+
+ ## Server to reroute to
+OVERFLOW_SERVER_URL byond://nanotrasen.se:6666
```

Comment:
This should probably start commented out with an example URL, like the other config URLs.